wandb/cli/cli.py

```diff
@@ -879,7 +883,7 @@ def _user_args_to_dict(arguments, argument_type="P"):
 @click.argument("uri")
 @click.option(
     "--entry-point",
-    "-e",
+    "-E",
```

Comment:
is there a reason for this? shouldn't this be `-e`

Reply:
entity is -e